includes/validation/class-amp-validation-manager.php

```diff
@@ -1930,53 +1931,88 @@ private static function update_admin_bar_item( Document $dom, $total_count, $kep
 		 */
 		if ( $kept_count > 0 && ! amp_is_canonical() ) {
 			$admin_bar_icon->setAttribute( 'class', 'ab-icon amp-icon ' . Icon::INVALID );
-		} elseif ( $unreviewed_count > 0 ) {
+		} elseif ( $unreviewed_count > 0 || $kept_count > 0 ) {
 			$admin_bar_icon->setAttribute( 'class', 'ab-icon amp-icon ' . Icon::WARNING );
 		}
 
-		$text = sprintf(
-			/* translators: %s is total count of validation errors */
-			_n(
-				'Validate %s issue',
-				'Validate %s issues',
-				$total_count,
-				'amp'
-			),
-			number_format_i18n( $total_count )
-		);
+		// Update the text of the link if there are validation errors.
+		if ( $unreviewed_count > 0 && $kept_count > 0 ) {
+			$text = sprintf(
+				/* translators: %s is total count of validation errors */
+				_n(
+					'%s issue:',
+					'%s issues:',
+					$total_count,
+					'amp'
+				),
+				number_format_i18n( $total_count )
+			);
```

Comment:
Maybe this should be omitted since it is not included when `$unreviewed_count > 0` or `$kept_count > 0`

Reply:
Or rather it should _always_ be included.